src/editor/index.html

```diff
@@ -309,8 +309,8 @@ <h3 id="layersLabel">Layers</h3>
         </div>
         <div id="text_panel">
           <div class="toolset">
-            <div class="tool_button" id="tool_bold" title="Bold Text [B]"><span></span>B</div>
-            <div class="tool_button" id="tool_italic" title="Italic Text [I]"><span></span>i</div>
+            <se-button id="tool_bold" title="Bold Text [B]" src="./images/bold.png"></se-button>
```

Comment:
se-button includes the ability for a "shortcut" attributes so you need to write: 
`<se-button id="tool_bold" title="Bold Text" shortcut="B" src="./images/bold.png"></se-button>`